### DISCUSSION TITLE
Doc comment on Keyword & Hybrid search (/developers/academy/py/starter_text_data/text_searches/keyword_hybrid)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1891] [Docs/blog comments]
# Doc comment on Keyword & Hybrid search (/developers/academy/py/starter_text_data/text_searches/keyword_hybrid)

You can also perform keyword (BM25) searches to find items based on their keyword similarity, or hybrid searches that combine BM25 and semantic/vector searches.

https://weaviate.io/developers/academy/py/starter_text_data/text_searches/keyword_hybrid

**Comment by homanmirgolbabaee** [comment]
Semantic Search 🐐
